modules/installation-vsphere-config-yaml.adoc

```diff
@@ -24,63 +24,94 @@ You can customize the `install-config.yaml` file to specify more details about
 your {product-title} cluster's platform or modify the values of the required
 parameters.
 
+[NOTE]
+====
+If you want to deploy an {product-title} cluster to multiple vSphere datacenters that run in a single VMware vCenter, you must modify the `install-config.yaml` file for this environment. For more information about creating this environment, see the section named _VMware vSphere region and zone enablement_. 
+====
+
 [source,yaml]
 ----
+additionalTrustBundlePolicy: Proxyonly
 apiVersion: v1
 baseDomain: example.com <1>
 compute: <2>
-- hyperthreading: Enabled <3>
-  name: worker
+- architecture: amd64
+  hyperthreading: Enabled <3>
+  name: <worker_node>
+  platform: {}
   replicas: 0 <4>
 controlPlane: <2>
+  architecture: amd64
   hyperthreading: Enabled <3>
-  name: master
+  name: <parent_node>
+  platform: {}
   replicas: 3 <5>
 metadata:
+  creationTimestamp: null
   name: test <6>
+networking:
+...
+...
 platform:
   vsphere:
-    vcenter: your.vcenter.server <7>
-    username: username <8>
-    password: password <9>
-    datacenter: datacenter <10>
-    defaultDatastore: datastore <11>
-    folder: "/<datacenter_name>/vm/<folder_name>/<subfolder_name>" <12>
-    resourcePool: "/<datacenter_name>/host/<cluster_name>/Resources/<resource_pool_name>" <13>
-    diskType: thin <14>
+    apiVIPs:
+    - 10.0.0.1
+    failureDomains:
+    - name: <failure_domain_name>
+      region: <default_region_name>
+      server: vcenters[0].server
+      topology:
+        computeCluster: "/<datacenter>/host/<cluster>"
+        datacenter: <datacenter> <7>
+        datastore: "/<datacenter>/datastore/<datastore>"
+        networks:
+        - <VM_Network_name>
+        resourcePool: "/<datacenter>/host/<cluster>/Resources/<resourcePool>" <8>
+        folder: "/<datacenter_name>/vm/<folder_name>/<subfolder_name>" <9>
+      zone: <default_zone_name>
+    ingressVIPs:
+    - 10.0.0.2
+    vcenters: <10>
+    - datacenters:
+      - <datacenter>
+      password: <password> <11>
+      port: 443
+      server: vcenters[0].server
+      user: administrator@vsphere.local
+      diskType: thin <12>
```

Comment:
I think diskType here is at the wrong level. Should be under
```
platform:
  vsphere:
    diskType: thin
```

Reply:
:heavy_check_mark: Indentation adjusted but position the same. Correct?